jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/Dispatcher.java

```diff
@@ -270,7 +270,7 @@ public String getQueryString()
         @Override
         public String getRequestURI()
         {
-            return _uri == null ? super.getRequestURI() : _uri.getPath();
+            return _uri == null ? super.getRequestURI() : _uri.getPathQuery();
```

Comment:
This is the wrong fix (sorry I know I suggested it).  `getRequestURI` should not include the query string.